index.bs

```diff
@@ -3635,6 +3801,35 @@ samesite-flag, which is from [[SAME-SITE-COOKIES]].
 
 </div>
 
+#### The network.CookieHeader Type #### {#type-network-CookieHeader}
+
+[=Remote end definition=]
+
+<pre class="cddl remote-cddl">
+network.CookieHeader = {
+    name: text,
+    value: network.BytesValue,
+};
+</pre>
+
+The <code>network.CookieHeader</code> type represents the subset of cookie data
+that's in a <code>Cookie</code> request header.
+
+<div algorithm>
+To <dfn>serialize cookie header</dfn> given |protocol cookie|:
+
+1. Let |name| be [=UTF-8 encode=] |protocol cookie|["<code>name</code>"].
+
+1. Let |value| be [=deserialize protocol bytes=] with |protocol
+   cookie|["<code>value</code>"].
+
+1. Let |header value| be the [=byte sequence=] formed by concatenating |name|,
```

Comment:
Do we need any special treatment for newline characters here (e.g. disallow, or escape)? Thinking about HTTP header injection.